apis/v1beta1/vspherecluster_types.go

```diff
@@ -59,6 +59,12 @@ type VSphereClusterSpec struct {
 	// +optional
 	ClusterModules []ClusterModule `json:"clusterModules,omitempty"`
 
+	// DisableClusterModule is used to explicitly turn off the ClusterModule feature.
+	// This should work along side NodeAntiAffinity feature flag.
+	// If the NodeAntiAffinity feature flag is turned off, this will be disregarded.
```

Comment:
this is fine but has to be updated once the feature flag graduates and is removed

Reply:
yeah. Will do it once it is graduated